core/frontend/src/cosmos.ts

```diff
@@ -15,7 +16,12 @@ declare global {
 
 // eslint-disable-next-line
 Array.prototype.first = function<T> (this: T[]): T | undefined {
-  return this.isEmpty() ? undefined : this[0]
+  return this[0]
+}
+
+// eslint-disable-next-line
+Array.prototype.last = function<T> (this: T[]): T | undefined {
+  return this.slice(-1)[0]
```

Comment:
Maybe we could use `at`

```suggestion
  return this.at(-1)
```

Reply:
Thanks!